File: src/main/java/com/iflat/pam/bean/Activist.java
package com.iflat.pam.bean;

import java.util.Date;

/**
 * 入党积极分子
 */
public class Activist {

    private String id;
    private String pbName;
    private String name;
    private String sex;
    private String dept;
    private Date birth;
    private Date applyTime;
    private Date becomeActivist;

    private String creatorAcc;
    private String creatorName;
    private Date createrTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPbName() {
        return pbName;
    }

    public void setPbName(String pbName) {
        this.pbName = pbName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    public String getDept() {
        return dept;
    }

    public void setDept(String dept) {
        this.dept = dept;
    }

    public Date getBirth() {
        return birth;
    }

    public void setBirth(Date birth) {
        this.birth = birth;
    }

    public Date getApplyTime() {
        return applyTime;
    }

    public void setApplyTime(Date applyTime) {
        this.applyTime = applyTime;
    }

    public Date getBecomeActivist() {
        return becomeActivist;
    }

    public void setBecomeActivist(Date becomeActivist) {
        this.becomeActivist = becomeActivist;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreaterTime() {
        return createrTime;
    }

    public void setCreaterTime(Date createrTime) {
        this.createrTime = createrTime;
    }
}

File: web/app/model/pam/Activist.js
Ext.define('iFlat.model.pam.Activist', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'activist.id', mapping: 'id', type: 'string'},
        {name: 'activist.pbName', mapping: 'pbName', type: 'string'},
        {name: 'activist.name', mapping: 'name', type: 'string'},
        {name: 'activist.sex', mapping: 'sex', type: 'string'},
        {name: 'activist.dept', mapping: 'dept', type: 'string'},
        {name: 'activist.birth', mapping: 'birth', type: 'date'},
        {name: 'activist.applyTime', mapping: 'applyTime', type: 'date'},
        {name: 'activist.becomeActivist', mapping: 'becomeActivist', type: 'date'},
        {name: 'activist.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'activist.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'activist.createrTime', mapping: 'createrTime', type: 'date'},
    ]
});